ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -56,13 +57,25 @@ import {
   createRunFormSchema,
   CreateRunFormValues,
   defaultValues,
+  flattenErrors,
   formValuesToPayload,
 } from './-create-run-utils';
 
 const CreateRunPage = () => {
   const navigate = useNavigate();
   const params = Route.useParams();
   const ortRun = Route.useLoaderData();
+  const [openAccordions, setOpenAccordions] = useState<string[]>([]);
+
+  // Manually toggle accordion open/close state
+  const handleAccordionChange = (value: string) => {
+    setOpenAccordions(
+      (prev) =>
+        prev.includes(value)
+          ? prev.filter((v) => v !== value) // Close accordion if open
+          : [...prev, value] // Open accordion if closed
+    );
+  };
```

Comment:
This could have a more descriptive name like toggleAccordionOpen or something like that.